hail/python/hail/table.py

```diff
@@ -1244,6 +1244,17 @@ def __repr__(self):
         def _repr_html_(self):
             return self.table._html_str(self.n, self.types)
 
+    def _take_n(self, n):
+        if n < 0:
+            rows = self.collect()
```

Comment:
I think of a negative number as the equivalent of `tail`. Since you're collecting all of the data anyways, i think you can implement this.

Reply:
we had these semantics before show() was rewritten in Python -- I'm reticent to change them now. I also don't want to implement showing the tail as collecting all the data -- people would try to use it and immediately run OOM.

I'm OK supporting a `tail=True` parameter on show() in the future, but we need to write an implementation that doesn't require localizing all the data.

Reply:
In that case, can we add something to the docs that states what a negative argument does?